trading/exchanges/rest_exchanges/rest_exchange.py

```diff
@@ -229,6 +231,33 @@ def create_order(self, order_type, symbol, quantity, price=None, stop_price=None
             self.logger.exception(e)
         return None
 
+    # todo { 'type': 'trailing-stop' }
+    async def _create_specific_order(self, order_type, symbol, quantity, price=None):
+        created_order = None
+        if order_type == TraderOrderType.BUY_MARKET:
+            created_order = await self.client.create_market_buy_order(symbol, quantity)
+        elif order_type == TraderOrderType.BUY_LIMIT:
+            created_order = await self.client.create_limit_buy_order(symbol, quantity, price)
+        elif order_type == TraderOrderType.SELL_MARKET:
+            created_order = await self.client.create_market_sell_order(symbol, quantity)
+        elif order_type == TraderOrderType.SELL_LIMIT:
+            created_order = await self.client.create_limit_sell_order(symbol, quantity, price)
+        elif order_type == TraderOrderType.STOP_LOSS:
+            created_order = None
+        elif order_type == TraderOrderType.STOP_LOSS_LIMIT:
+            created_order = None
+        elif order_type == TraderOrderType.TAKE_PROFIT:
+            created_order = None
+        elif order_type == TraderOrderType.TAKE_PROFIT_LIMIT:
+            created_order = None
+        return created_order
+
+    @staticmethod
+    def _ensure_order_details_completeness(order):
+        order_required_fields = [ecoc.ID.value, ecoc.TIMESTAMP.value, ecoc.SYMBOL.value, ecoc.TYPE.value,
+                                 ecoc.SIDE.value, ecoc.PRICE.value, ecoc.AMOUNT.value, ecoc.REMAINING.value]
+        return all(key in order for key in order_required_fields)
```

Comment:
Why using a local variable ?

Reply:
ease to read

Reply:
It didn't change anything because `order_required_fields` is always `None`.
I think a class variable can be better.

Reply:
Github is outdated, here is the code:
```python
    @staticmethod
    def _ensure_order_details_completeness(order, order_required_fields=None):
        if order_required_fields is None:
            order_required_fields = [ecoc.ID.value, ecoc.TIMESTAMP.value, ecoc.SYMBOL.value, ecoc.TYPE.value,
                                     ecoc.SIDE.value, ecoc.PRICE.value, ecoc.AMOUNT.value, ecoc.REMAINING.value]
        return all(key in order for key in order_required_fields)
```

Reply:
Here the function argument is changed during the 1st call, therefore initalized once only

Reply:
Ok great !